tcms/core/management/commands/upgrade.py

```diff
@@ -0,0 +1,66 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Perform post-upgrade tasks for Kiwi TCMS"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--noinput",
+            "--no-input",
+            action="store_false",
+            dest="interactive",
+            help="Automatic mode. Does not require user confirmation",
+        )
+
+    def handle(self, *args, **kwargs):
+        answer = "y"
+        if kwargs["interactive"]:
+            answer = "x"
+
+        self.stdout.write(
+            """To finish the upgrade process, the following
+management commands will be executed:
+
+migrate
+refresh_permissions
+clean_orphan_obj_perms
+remove_stale_contenttypes
```

Comment:
`clean_orphan_obj_perms`, and `remove_stale_contenttypes` are both executed as part of `refresh_permissions`. No need to execute them again.